src/client.cpp

```diff
@@ -2199,13 +2199,13 @@ void Client::sendChangePassword(const std::wstring &oldpassword,
 }
 
 
-void Client::sendDamage(u8 damage)
+void Client::sendDamage(s8 damage)
 {
 	DSTACK(__FUNCTION_NAME);
 	std::ostringstream os(std::ios_base::binary);
 
 	writeU16(os, TOSERVER_DAMAGE);
-	writeU8(os, damage);
+	writeS8(os, damage);
```

Comment:
Range of signed 8bit integer should be -128 to 127, so it's true that it is small range. This integer is for sending any damage to server, so if I'm right if somebody damages you >128 HP, he heals you. Protocol compatibility is probably already broken, so I am going to change it to s16, that's -32,768 to 32,767 damage per attack.